packages/design-system/src/components/SearchBar/style.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../scss/variables/colors';
+
+$base-class: "search-bar";
+
+// TODO: should container have fixed width and height?
```

Comment:
it should have height. Width should be 100%. It's pretty common now that you need to add styles to input to use it in AA because it doesn't have `width: 100%`.

Reply:
I've added `width: 100%`, but I've got a question. Should I leave `max-width: 290px` or nope? Because right now my component spans for 100% width and it doesn't look good. Or maybe it should stay for 100% width and lets user wrap it inside some component and style it on their own when importing this component from design system?

Reply:
In other libraries, there is no max-width for input components. Let's leave it like that.